ydb/public/lib/ydb_cli/commands/ydb_latency.cpp

```diff
@@ -143,8 +146,13 @@ TCommandLatency::TCommandLatency()
     , Format(DEFAULT_FORMAT)
     , RunKind(DEFAULT_RUN_KIND)
     , Percentile(DEFAULT_PERCENTILE)
+    , ChainConfig(new NDebug::TActorChainPingSettings())
 {}
 
+TCommandLatency::~TCommandLatency() {
+    delete ChainConfig;
```

Comment:
😨

Reply:
Otherwise I either need to include everything in the header or to rewrite the class using pimpl. Both ways are ugly in this particular case, imho, while this one is slightly less ugly, but fast.

Reply:
```cpp
namespace NDebug {
    class TActorChainPingSettings;
}
...
std:unique_ptr<NDebug::TActorChainPingSettings> ChainConfig;
```